transaction/memo.go

```diff
@@ -0,0 +1,144 @@
+package transaction
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/gagliardetto/solana-go"
+	solanarpc "github.com/gagliardetto/solana-go/rpc"
+)
+
+const BxMemoMarkerMsg = "Powered by bloXroute Trader Api"
+
+var TraderAPIMemoProgram = solana.MustPublicKeyFromBase58("HQ2UUt18uJqKaQFJhgV9zaTdQxUZjNrsKFgoEDquBkcx")
+
+// CreateTraderAPIMemoInstruction generates a transaction instruction that places a memo in the transaction log
```

Comment:
add more detail to the to the comment to indicate this is required for submitting transactions through Trader API